content/introduction/getting-started-with-sdks/index.mdx

```diff
@@ -0,0 +1,303 @@
+---
+title: Getting started with the JavaScript SDKs
+description:
+  This article serves as a guide on how to get started using
+  Sharetribe's JavaScript SDKs.
+---
+
+import { Callout } from 'nextra/components';
+import { Table } from 'nextra/components';
+
+# Getting started with the JavaScript SDK
+
+Sharetribe's JavaScript Software Development Kit (SDK) and Integration
+SDK simplify interactions with our APIs. You can read more about
+features and functionalities in our
+[SDK GitHub Documentation](https://sharetribe.GitHub.io/flex-sdk-js/index.html).
+This guide covers:
+
+- Installation
+- Initialization and instantiating the SDK
+- Creating a trusted SDK instance
+- Example use cases
+
+## Overview
+
+Our SDKs simplify making requests to the backend by handling
+authentication and providing convenient access to Sharetribe's APIs.
+They allow you to interact easily with:
+
+- Marketplace API, for general interactions with marketplace data
+- Asset Delivery API, for delivering assets configured in Console
+- Integration API, for advanced operational use
+
+You can find more information on API usage and guidelines for choosing
+which to work with in
+[this article](https://www.sharetribe.com/docs/concepts/api-sdk/marketplace-api-integration-api/#when-to-use-the-marketplace-api).
+
+There are two SDK variants in which to access the APIs:
+
+<br />
+<Table>
+  <thead>
+    <Table.Tr>
+      <Table.Th />
+      <Table.Th>Sharetribe SDK</Table.Th>
+      <Table.Th>Sharetribe Integration SDK</Table.Th>
+    </Table.Tr>
+  </thead>
+  <tbody>
+    <Table.Tr>
+      <Table.Td>Intended Use</Table.Td>
+      <Table.Td>
+        Most common API requests with Marketplace API or Asset Delivery
+        API
+      </Table.Td>
+      <Table.Td>Operator-level requests with Integration API</Table.Td>
+    </Table.Tr>
+    <Table.Tr>
+      <Table.Td>Required Parameters</Table.Td>
+      <Table.Td>Client ID</Table.Td>
+      <Table.Td>Client ID, Client secret</Table.Td>
+    </Table.Tr>
+    <Table.Tr>
+      <Table.Td>Secure context needed</Table.Td>
+      <Table.Td>No</Table.Td>
+      <Table.Td>**Yes**</Table.Td>
+    </Table.Tr>
+  </tbody>
+</Table>
+
+To try out the Sharetribe SDK, visit our
+[Github documentation](https://sharetribe.GitHub.io/flex-sdk-js/index.html)
+for instructions on testing it directly in your browser or via the
+[API Playground](https://sharetribe.GitHub.io/flex-sdk-js/try-it-in-the-playground.html).
+For a more detailed guide on using the SDK with code, keep following
+this tutorial.
+
+## Installation with Yarn Package Manager
+
+To get started with the SDK, you'll first need to install
+[Yarn Package Manager](https://yarnpkg.com/). After that is set up, you
```

Comment:
Would it make sense to offer both yarn and npm as options here? Since folks are using their own clients, they might already be using npm and not want to install yarn for just this use case.